virt/virtual_machines/virt-vtpm-devices.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices_{context}"]
+= Virtual trusted platform module devices
+include::_attributes/common-attributes.adoc[]
+:context: virt-vtpm-devices
+
+toc::[]
+
+:FeatureName: Virtual trusted platform module devices
+
+A Virtual Trusted Platform Module (vTPM) device protects virtual machines
+(VMs) by storing secrets similarly to a physical Trusted Platform Module (TPM)
+hardware chip.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
```

Comment:
I think there are too many `[NOTE]` blocks in this short assembly (and two of them are the same). I can see where you're coming from, but I think most of this information might be better in the body text.